src/customize-pop/accessibility-seeing.md

```diff
@@ -0,0 +1,187 @@
+# Seeing
+
+`Seeing` settings provide visual enhancements to help visually impaired users navigate the operating system's user interface.
+
+## High Contrast
+
+Enabling `High Contrast` applies a high contrast theme to windows and icons.
+
+### High Contrast Mode Off
+
+**Click image to enlarge**
+
+<a href="/images/accessibility-settings/contrast-mode-off.png" target="blank">
+    <img src="/images/accessibility-settings/contrast-mode-off.png" width="733px" height="456px">
+</a>
+
+### High Contrast Mode On
+
+**Click image to enlarge**
```

Comment:
The repeated notes about enlarging look cluttered. I kind of understand since this is an accessibility section. Do we think the user really needs to see individual elements in these screenshots, as opposed to just the colors, to get the general idea across?

Reply:
These resolved an issue raised in Daniel's feedback for this section. The concern was that GNOME's current documentation doesn't provide these images, and enlarging the images allows visually impaired users to see both the option's toggle switch and its effects. 

![image](https://user-images.githubusercontent.com/80931038/177416101-826d5ce8-8ff2-45ae-ab8c-91f4da8f6073.png)

I agree the repeated prompts visually conflict with headers and it doesn't look great. I also don't like that clicking the image takes the user to a new place, but as a reader I always prefer things open in a new tab vs. having to navigate back to what I was reading. 

- Change the text formatting of the prompts to improve their visual aesthetic (simple)
- Implement a better "zoom" capability (complex), [maybe something like this](https://www.geeksforgeeks.org/how-to-make-the-images-bigger-when-clicked/)

Reply:
If someone needs to zoom in to see better, they can `Ctrl` + `+` or `Ctrl` + scroll up. Perhaps a note about the default zoom shortcuts for most web browsers at the top of the page would suffice?

Also, just playing devil's advocate, the user has to navigate to this page before they get the `Click image to enlarge` prompts anyway.

I think providing usable alt text for screen readers (which you've generally been doing, but are missing on these images since things have now gotten distracted by custom HTML) is going to be more helpful than implementing special click-to-zoom functionality.

Reply:
Woah, that's a crazy solution.

https://user-images.githubusercontent.com/7199422/177430469-c1f745b5-5fd7-428c-b2dc-4b29f8c00a10.mp4

I don't hate it, but it could be a little overwhelming. If we go this route, I at least think the image should not expand beyond the width of the window/pane. It looks like this has also broken resizing-- the hoverable images are getting horizontally squished now:

https://user-images.githubusercontent.com/7199422/177430610-ef71056b-bb8e-4e52-96e9-f2bf9b0ad0a5.mp4

Reply:
Oops, the aspect ratio issue occurred because I defined height and width in the `img src` tag, and removing those fixed that issue.

I added a media query to the css class that requires a minimum browser window size (HxW) for the effect to occur. These images are all similar in size, so this works for these images for this section. These parameters also appear to apply appropriately for various browser zoom levels.

Reply:
With the current width restriction, I'm not opposed to including this as an accessibility feature on this page only. We can remove it later if the UX doesn't end up working for people.